rest-api/test/unit_test/database_test.py

```diff
@@ -28,9 +29,11 @@ def test_schema(self):
 
     hpo = HPO(hpoId=1, name='UNSET', displayName='No organization set',
               organizationType=OrganizationType.UNSET)
+    calendar = Calendar(day=datetime.date(2018, 01, 01))
```

Comment:
you can just do 1 and 1 here... 01 looks weird in github UI. :)

Reply:
Done.